ci-operator/config/openshift/openshift-tuned/openshift-openshift-tuned-master.yaml

```diff
@@ -0,0 +1,41 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: origin-v4.0
+    namespace: openshift
+    tag: base
+canonical_go_repository: github.com/openshift/openshift-tuned
+images:
+- from: base
+  inputs:
+    root:
+      as:
+      - '0'
+  to: openshift-tuned
```

Comment:
Your name should be `cluster-tuning-operator` or similar.

Reply:
@smarterclayton, cluster-node-tuning-operator will follow, this PR is just for the containerized tuned image, which the https://github.com/openshift/cluster-node-tuning-operator will use.

Reply:
If this is an image used by that operator, then it shouldn't have "openshift" as a prefix.  Everything inside our release is destined for openshift, so the name is too generic.  Call this `cluster-node-tuned`

Reply:
Addressed in https://github.com/openshift/release/pull/2007